apps/i18n/aichat/en_us.json

```diff
@@ -2,5 +2,11 @@
   "continue": "Continue",
   "aichatWorkspaceHeader": "AI Chat",
   "inappropriateUserMessage": "This message has been flagged as inappropriate.",
-  "tooPersonalUserMessage": "This message has been flagged as too personal."
+  "tooPersonalUserMessage": "This message has been flagged as too personal.",
+  "warningModalHeader": "Remember to chat responsibly!",
+  "allMessagesSentAreRecorded": "All messages sent to AI Chat are recorded.",
+  "inappropriateMessagesAreFlagged": "Inappropriate messages will be flagged and your teacher will be notified. Users who send more than X inappropriate messages will be locked out of Chat AI.",
```

Comment:
Also, I think the name of app is 'AI Chat' instead of 'Chat AI'. I'll ask Sam to confirm this.

Reply:
This is a question for Sam as well...maybe instead of 'anything deemed personal information', we could use 'anything considered personal information'.

Reply:
Yup! The name is "AI chat". 

Lets change the last sentence to: In order to protect your privacy, anything considered personal information (name, phone number, address, etc.) should not be submitted to AI chat. 

Also we can remove the sentence: users who send more than X...... 
That is a feature for v1 and we dont have it implemented for the MVP, so we can update the dialog when we add that feature.

Reply:
Hi Denys - wrt to my comment about the `inappropriateMessageLimit` constant - I apologize for being unclear. See Slack message [here](https://codedotorg.slack.com/archives/C05L3RZJZ97/p1692122552327039?thread_ts=1692106703.913609&cid=C05L3RZJZ97) - thanks!

Reply:
Gotcha, thank you both for the info!
Will update